lib/tai/advisors/factories/shared_venues_and_shared_products.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Tai.Advisors.Factories.SharedVenuesAndSharedProducts do
```

Comment:
How do you feel about calling this `OneForAllProducts`?

Reply:
Just out of curiosity, what are some other factories did you have in mind that aren't here yet?

Reply:
Some of the ones that I feel comfortable making public are:

- pairs between venues
- triangular within & between venues
- first N